selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -99,15 +99,21 @@ void WifiManager::refreshFinished(QDBusPendingCallWatcher *watcher) {
     auto properties = replay.value();
 
     const QByteArray ssid = properties["Ssid"].toByteArray();
-    uint32_t strength = properties["Strength"].toUInt();
-    if (ssid.isEmpty() || (seenNetworks.contains(ssid) && strength <= seenNetworks[ssid].strength)) continue;
```

Comment:
@deanlee do you know how `if (path.path() == activeAp)` worked below? Is it true for all access points with the ssid, or only the one you're connected to?